install-concept.md

```diff
@@ -0,0 +1,50 @@
+1. Bootstrap host in e. g. CSP hidden Openstack project, install kind+CAPI there
+```mermaid
+flowchart LR
+    A[CSP Bootstrap host] -->|install kind+CAPI| A
+```
+
+2. Use bootstrapping kind+CAPI to create Mgmt cluster (in e. g. CSP hidden Openstack project using application credentials)
+```mermaid
+flowchart LR
+    A[CSP Bootstrap host with kind+CAPI] -->|Create| B(CSP CAPI mgmt cluster)
+```
+
+3. `clusterctl move` to new cluster, which should start managing itself
```

Comment:
From CAPI perspective you create a bootstrap cluster with kind, then a workload cluster (i.e. a proper Kubernetes cluster) and then move the management components from the bootstrap cluster in there. I feel like the current state is a bit misleading in this regard ;)

Reply:
Reworded this a bit and added multiple references to upstream docs. Do you think that improved the situation?